### PR TITLE
Shop page

### PR DESCRIPTION
**Pull request by VedanthB**
- added sshop page
- added shop page components

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,18 +1,33 @@
...
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/shop" element={<Shop />} />
         <Route path="/cart" element={<Cart />} />
-        <Route path="wishlist" element={<Wishlist />} />
-        <Route path="login" element={<Login />} />
+        <Route path="/wishlist" element={<Wishlist />} />
+        <Route path="/login" element={<Login />} />
         <Route path="/signup" element={<Signup />} />
       </Routes>
```
Comment (gopalbbharadva) [inline src/App.js:27]: Suggestion: You can make a separate component for routes, so Your App component will looks cleaner.

File: src/routes/Home.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Banner from "../components/home-page/Banner";
+import FeaturedBrands from "../components/home-page/FeaturedBrands";
+import FeaturedCategories from "../components/home-page/FeaturedCategories";
+import VideoBanner from "../components/home-page/VideoBanner";
+
+function Home() {
+  return (
+    <div className="relative" style={{ top: "5rem" }}>
```
Comment (gopalbbharadva) [inline src/routes/Home.js:9]: Suggestion, remove inline styles, may be you forgot.

File: src/components/home-page/FeaturedBrands.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import santaCruzImg from "../../assets/santa-cruz-brand.png";
+import urbanMonkeyImg from "../../assets/urban-money-brand.png";
+import vansImg from "../../assets/vans-brand.png";
+import thrasherImg from "../../assets/thrasher-brand.png";
```
Comment (gopalbbharadva) [inline src/components/home-page/FeaturedBrands.js:6]: Suggestion : you can make barrel export for images where you export all the images in one file and import it from one file at here.

P.S: You can do barrel export for components ,pages, contexts etc..

**Review by gopalbbharadva — APPROVED** [review]
Code looks clean, and structured.
one suggestion : at many place you used inline styles, you can avoid that if it's not intentional.

File: src/App.js [hunk 4]
```diff
@@ -1,41 +1,33 @@
...
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      {/* header / nav */}
+      <Header />
+      {/* routes */}
```
Comment (tanishk15sharma) [inline src/App.js:19]: You can remove this comment

File: src/components/global/Footer.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+function Footer() {
+  return (
+    <footer
+      style={{ bottom: 0 }}
+      className="relative flex flex-col h-44 justify-center align-items-center w-full p-5 z-50 bg-regal-blue-dark"
+    >
+      <div className="text-grey-700">
+        Made with
+        <span className="text-cyan-700 font-bold ml-1 mr-1">&lt;/&gt; </span> by
+        <a
```
Comment (tanishk15sharma) [inline src/components/global/Footer.js:12]: Avoid using `<a>` tag in react , use can use `<Link>` tag

File: src/components/global/Footer.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+        </a>
+        <a href="https://dev.to/vedanthb" target="_blank" rel="noreferrer">
+          <i className="fab fa-dev text-cyan-700"></i>
+        </a>
+      </div>
+      <div className="text-grey-700 mt-3">
+        Designed using
+        <a
+          href="https://argon-css.netlify.app/"
+          style={{ padding: 0, textTransform: "none" }}
```
Comment (tanishk15sharma) [inline src/components/global/Footer.js:46]: Avoid using `inline` style

File: src/routes/Shop.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    >
+      <FiltersSideBar />
+
+      <div className="products-display-container">
+        <div className="flex w-full justify-end">
+          <SortProductsDropdown />
+        </div>
+
+        <div style={{ gap: "4rem" }} className="grid grid-cols-3">
+          {/* <ProductCard product={{}} />
```
Comment (tanishk15sharma) [inline src/routes/Shop.js:20]: You can remove these comments as well (line no. 20 to 26)

**Review by tanishk15sharma — APPROVED** [review]
You code is good to merge , you can follow the suggestions give above .

File: src/App.js (hunk 1 above)
Comment (VedanthB) [reply src/App.js:27]: yup that would look cleaner , will implement in the next PR

File: src/routes/Home.js (hunk 2 above)
Comment (VedanthB) [reply src/routes/Home.js:9]: No , these are to overwrite the component lib class styles

File: src/components/home-page/FeaturedBrands.js (hunk 3 above)
Comment (VedanthB) [reply src/components/home-page/FeaturedBrands.js:6]: oh i didnt think of that, i will implement it in the next PR

File: src/App.js (hunk 4 above)
Comment (VedanthB) [reply src/App.js:19]: Okay i will remove the comments

File: src/components/global/Footer.js (hunk 5 above)
Comment (VedanthB) [reply src/components/global/Footer.js:12]: oops! , forgot to change them, will change it to link

File: src/components/global/Footer.js (hunk 6 above)
Comment (VedanthB) [reply src/components/global/Footer.js:46]: okay

File: src/routes/Shop.js (hunk 7 above)
Comment (VedanthB) [reply src/routes/Shop.js:20]: okay i will remove the commented code

**Pull request merged**